spec/consensus/reactor/reactor.md

```diff
@@ -1,196 +1,141 @@
-The first layer, **CONS**, keeps the state and transition functions described in the [Tendermint BFT paper][1].
-Instances of CONS use gossiping to communicate with other nodes.
-The second layer, **GOSSIP**, keeps state and transition functions needed to implement gossiping on top of the 1-to-1 communication facilities provided by the P2P layer.
-Exchanges between CONS and GOSSIP use multiple forms, but we will call them all **GOSSIP-I** here.
-
-
-```
-...
-==========ABCI=========
-                         ┐
-  |      CONS       |    |
-  |.....GOSSIP-I....|    |  Consensus Reactor
-  |     GOSSIP      |    |
-                         ┘
-- - - - - P2P-I - - - -
-         P2P
-=======================
-    Network Stack
-```
-
-The goals of this document are to specify the following:
-* what CONS requires from and provides to GOSSIP (GOSSIP-I);
-* what GOSSIP requires from and provides to CONS (GOSSIP-I); and,
-* GOSSIP requires from and provides to P2P (P2P-I) in order to satisfy CONS' needs.
-
-The specification is divided in multiple documents
-* reactor.md (this document): specification in English
-* [reactor.qnt](./reactor.qnt): corresponding specifications in [Quint](https://github.com/informalsystems/quint)
-* [implementation.md](./implementation.md): a description of what is currently implemented in Tendermint Core, in English.
-* [implementation.qnt](./implementation.qnt): Quint model of current behavior, for model checking of provided properties.
-
-# Conventions
+# Part 1: Background
 
-* MUST, SHOULD, MAY...
-* [X-Y-Z-W.C]
-    * X: What
-        * VOC: Vocabulary
-        * DEF: Definition
-        * REQ: Requires
-        * PROV: Provides
-    * Y-Z: Who-to whom
-    * W.C: Identifier.Counter
+> **Warning**    
+> We assume that you understand the Tendermint algorithm and therefore will not review it here. If this is not the case, please refer to [here](../).
 
+The Tendermint algorithm assumes that a **Global Stabilization Time (GST)** exists, after which communication is reliable and timely. That is, it satisfies **Eventual $\Delta$-Timely Communication**:
 
-# Status
+| Eventual $\Delta$-Timely Communication|
+|-----|
+|There is a bound $\Delta$ and an instant GST (Global Stabilization Time) such that if a correct process $p$ sends a message $m$ at a time $t \geq \text{GST}$ to a correct process $q$, then $q$ will receive $m$ before $t + \Delta$.
 
-> **Warning**    
-> This is a Work In Progress
+The Tendermint algorithm also assumes that this property is used to provide **Gossip Communication**:
 
-> **Warning**    
-> Permalinks to excerpts of the Quint specification are provided throughout this document for convenience, but may outdated.
+|Gossip communication|
+|-----|
+| (i) If a correct process $p$ sends some message $m$ at time $t$, all correct processes will receive $m$ before $\text{max} \{t,\text{GST}\} + \Delta$.
```

Comment:
Suggestion: would these definitions still be correct if we worded like this ...?

(i) If a correct process $p$ sends some message $m$ at time $t$, at least one correct processes will receive $m$ before $\text{max} \{t,\text{GST}\} + \Delta$.

Reply:
Seems correct. Any proofs of termination would have to consider $2\Delta$ as the actual deadline for decision, that's the only consequence, I think.
I wouldn't change the wording here, though, as this has been copied from the paper.